c10/util/constexpr_string_functions.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <cstdlib>
+
+#include <c10/util/string_view.h>
+
+namespace c10 {
+namespace util {
+
+// returns true iff whitelist contains item
+// op_whitelist_contains("a;bc;d", "bc") == true
+constexpr bool op_whitelist_contains(string_view whitelist, string_view item) {
```

Comment:
This is a pretty op-whitelist-specific piece of functionality to be in `c10/util/constexpr_string_functions.h`, no? Why not just put it in `op_whitelist.h`?

OTOH if you're thinking it's the first of a more general-purpose crop of utilities, I think we'd want to rename it to like `semi_delimited_list_contains` (and arguably just put it in `string_view.h`, seems like there's a good amount of this kind of stuff in there already)

Reply:
There's no good reason; @smessmer's original patch had this in its own header and I've been to lazy to move it.